.github/workflows/build-flatpack.yml

```
@@ -0,0 +1,56 @@
+name: Flatpak build - Linux
+
+on:
+  push:
+    branches:
+      - 'release/**'
+      - 'hotfix/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-22.04
```
GitHub just obsoleted the old runner image.